### PR TITLE
new interface filtering

### PR DESCRIPTION
**Pull request by benhaynes**

### CODE REVIEW

File: src/components/field-setup.vue [hunk 1]
```diff
@@ -1063,6 +1036,18 @@ export default {
     this.activeTab = this.existing ? "schema" : "interface";
   },
   methods: {
+    interfaceSubtitles(ext) {
+      if (ext.types) {
+        return this.$helpers.formatTitle(ext.types[0]);
+      } else {
+        return "String";
+      }
+      // if(ext.options){
```
Comment (rijkvanzanten) [inline src/components/field-setup.vue]: No commented out code

File: src/components/field-setup.vue [hunk 2]
```diff
@@ -637,89 +665,34 @@ export default {
...
       return groups.map(group => ({
         ...group,
         interfaces: group.interfaces.map(name => this.interfaces[name])
       }));
     },
+    interfacesFiltered() {
+      if (!this.interfaceFilter) return this.interfaces;
+      return Object.keys(this.interfaces)
+        .filter(interfaceName => interfaceName.includes(this.interfaceFilter))
+        .map(interfaceName => ({ ...this.interfaces[interfaceName] }));
```
Comment (rijkvanzanten) [inline src/components/field-setup.vue]:
```suggestion
        .map(interfaceName => this.interfaces[interfaceName]);
```

File: src/components/field-setup.vue [hunk 3]
```diff
@@ -637,89 +665,34 @@ export default {
     interfaces() {
       return Object.assign({}, this.$store.state.extensions.interfaces);
     },
-    interfacesGrouped() {
+    interfacesPopular() {
       const groups = [
```
Comment (rijkvanzanten) [inline src/components/field-setup.vue]: If these are just the popular ones, lets get rid of this groups abstraction

File: src/components/field-setup.vue (hunk 1 above)
Comment (benhaynes) [reply src/components/field-setup.vue]: All set.

Comment (benhaynes) [reply src/components/field-setup.vue]: Was just getting that other option into the git history ;)

File: src/components/field-setup.vue (hunk 3 above)
Comment (benhaynes) [reply src/components/field-setup.vue]: That's beyond my skillset. I'll leave this optimization to you... then we can merge. 😄

**Pull request merged**